.circleci/config.yml

```diff
@@ -182,9 +182,7 @@ jobs:
       - persist_to_workspace:
           root: bin/
           paths:
-            - artifacts/*.zip
-            - artifacts/*.tgz
-            - artifacts/*.tar
+            - artifacts/*
```

Comment:
We won't. This is only persisting to the workspace so that we can actively copy out the specific *.tgz/*.zip files we want. It's easier to add the files, but filter the upload - as opposed to grab just specific files and play.

This is how I discovered the symlink issue.